src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2404,6 +2417,7 @@ because xbuild doesn't support framework reference assemblies.
   <CreateItem
     Include="$(IntermediateOutputPath)android\src\\**\*.java">
     <Output TaskParameter="Include" ItemName="_JavaStubFiles" />
+    <Output TaskParameter="Include" ItemName="FileWrites" />
```

Comment:
For my knowledge:

If the target is skipped here, will these java files still get added to `FileWrites`? Does MSBuild special case `CreateItem` and `CreateProperty` tasks so they run?

Reply:
It should.. Will need to check. 

btw looks like `CreateItem` is deprecated [1] we should look at removing them.


[1] https://docs.microsoft.com/en-us/visualstudio/msbuild/createitem-task?view=vs-2017

Reply:
ok read up on https://docs.microsoft.com/en-us/visualstudio/msbuild/incremental-builds?view=vs-2017#output-inference.

The `Output` values (and `ItemGroup` and `PropertyGroup`) are all evaluated even if a target is skipped.